### DISCUSSION TITLE
Error exporting database when downloading relations.

### DISCUSSION THREAD

**Discussion by 2snEM6** [prisma/prisma#2104] [Q&A] [answered]
I have a problem when trying to export a Prisma database using `prisma export`.

```
client Downloading from <endpoint> +134ms
  client {"fileType":"relations","cursor":{"table":0,"row":0,"field":0,"array":0}} +0ms
  Exporter { errors:
  Exporter    [ { message: 'No such column :A',
  Exporter        requestId: '<request_id>' } ] } +140ms
 !
```

And I don't know how to solve it. Has anyone had a similar issue already and knows how to sort it out?

Thank you,

Daniel.

**Comment by nikolasburk** [comment, marked as answer]
Hey Daniel, the `prisma export` command is only available in the [Prisma 1 CLI](https://www.prisma.io/docs/1.34/prisma-cli-and-configuration/), but not yet supported in Prisma 2.0. You can import and export the data directly on the DB level, we have guides for that in the [docs](https://www.prisma.io/docs/), e.g. for [PostgreSQL](https://www.prisma.io/docs/guides/database-workflows/import-and-export-data/postgresql).

If that solves answers your question, please make sure to mark this comment as an answer 🙂

**Reply by 2snEM6** [reply]
Hi @nikolasburk thank you for your reply. My bad, didn't know the repo was renamed, and this error happens on Prisma 1,  should I move this discussion to the old repo? Thank you.

**Reply by janpio** [reply]
Yes please. We unfortunately can not convert discussions to issues (which we could then transfer to the correct repo).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
